backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -201,14 +202,16 @@ internal val psiToIrPhase = konanUnitPhase(
 
             val functionIrClassFactory = BuiltInFictitiousFunctionIrClassFactory(
                     symbolTable, generatorContext.irBuiltIns, reflectionTypes)
-            // TODO: Add special handling for enums and structs.
-            val irProviderForInteropStubs = IrProviderForInteropStubs { false }
+            val irProviderForCEnums = IrProviderForCEnumStubs(generatorContext, interopBuiltIns)
```

Comment:
Compiling an interop library to a cache probably requires generating IR for all enum classes.
See `functionIrClassFactory.buildAllClasses()` as an example below.